app/pods/components/web-components/au-icon/template.hbs

```diff
@@ -1,3 +1,3 @@
 <div class="auk-icon {{this.size}}">
-  <i class="ki ki-{{@name}} {{this.grab}}"></i>
+  <i class="ki ki-{{@name}} {{this.grab}}" style={{this.color}}></i>
```

Comment:
Voor een uitzondering kan 't in principe wel, maar weet niet of 't nodig is. Was er een technische horde om niet via (de voorziene?) css class (modifiers) te werken?

Reply:
dan moeten we voor elke icoon dat een andere kleur heeft een regel css schrijven.
Ben ook wel voorstander van `this.color`, en dan `ki-icon--color-warning` ofzo

Reply:
Ik zie het inderdaad liever ook niet op deze manier.. als er specifieke icon's een andere kleur moeten hebben dat dient dat op de gepast manier overruled te worden door de css klassen later in de reeks te zetten. 

I agree met Sven maar ik zou het als volgt doen: `ki-icon-color--warning`.

Best afstemmen met @Wolfr hoe de icon colors gebruikt moeten worden.. in mijn ogen is het een modifier maar is enkel het `warning` gedeelte het stuk van de modifier omdat enkel dat veranderd.. het feit dat de kleur veranderd is deel van de icon.